recipes/likwid/build.sh

```diff
@@ -0,0 +1,14 @@
+# Detect whether gcc or clang
+if [[ $CC == *"clang"* ]]; then
+    export COMPILER="CLANG"
+else
+    export COMPILER="GCC"
+fi
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    CUDA_HOME=${PREFIX}/pkgs/cuda-toolkit make CC=${CC} CXX=${CXX} PREFIX=${PREFIX} NVIDIA_INTERFACE=true ACCESSMODE=perf_event COMPILER=$COMPILER -j${CPU_COUNT}
```

Comment:
CUDA_HOME should already be set if you have acuda compiler. The path you use here looks awkward to me.